modules/preview-react/breadcrumbs/stories/Breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,219 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+import {Breadcrumbs} from '@workday/canvas-kit-preview-react/breadcrumbs';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {StandardList} from './examples/StandardList';
+import {CollapsibleList} from './examples/CollapsibleList';
+import {RTLCollapsibleList} from './examples/RTL';
+import {CurrentItemTruncation} from './examples/CurrentItemTruncation';
+import {LinkTruncation} from './examples/LinkTruncation';
+
+<Meta title="Preview/Breadcrumbs/React" component={Breadcrumbs} />
+
+# Canvas Kit Breadcrumbs
+
+Breadcrumbs allow users to keep track and maintain awareness of their location as they navigate
+through pages, folders, files, etc.
+
+[> Workday Design Reference](https://design.workday.com/components/navigation/breadcrumbs)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+Breadcrumbs is a composable component that provides some flexibility in it's structure. There are
+two main variations: standard list and collapsible list. We'll discuss the standard list first. All
+subcomponents are described in more detail in the [Components](#components) section below.
+
+### Standard List Example
+
+`Breadcrumbs.List` is a styled unordered list `<ul>` with no additional behaviors. Use this when you
+are not concerned about collapsing the list of breadcrumb items into a dropdown menu at a specified
+width.
+
+Accessibility is built into Breadcrumbs components in a way that allows you to create an inclusive
+experience with little additional configuration. The only ARIA label you'll need to add is the
+`aria-label` for `Breadcrumbs.Nav` as seen in the example. And all links require `href`s to be
+properly identified. All other accessible attributes are baked into the components.
+
+<ExampleCodeBlock code={StandardList} />
+
+### Collapsible List Example
+
+`Breadcrumbs.CollapsibleList` is built on top of `Breadcrumbs.List`. It provides additional
+functionality to collapse list items into a dropdown menu at a specified width.
+
+Accessibility is built into Breadcrumbs components in a way that allows you to create an inclusive
+experience with little additional configuration. The only ARIA labels you'll need to add are the
+`aria-label` for `Breadcrumbs.Nav` and the `buttonAriaLabel` for `Breadcrumbs.CollapsibleList`. And
+all links require `href`s to be properly identified. All other accessible attributes are baked into
+the components.
```

Comment:
Nit: could shorten this paragraph given that it's almost identical word-for-word to the paragraph in the `Standard List Example` section. Something like this:

```suggestion
In addition to adding `aria-label` to `Breadcrumbs.Nav` and identifying all links with an `href`,
you'll need to add `buttonAriaLabel` to `Breadcrumbs.CollapsibleList`.
```